frontend/src/components/form/api/ApiWrapper.vue

```diff
@@ -108,11 +108,19 @@ export default {
       // return value from API unless `value` is set explicitly
       } else {
         let val = this.api.get(this.uri)[this.fieldname]
+
+        // while loading, value is null
+        // (necessary because while loading, even normal properties are returned as functions)
+        if (val.loading) return null
```

Comment:
```suggestion
        if (val && val.loading) return null
```
I still get errors on reloading `/profile` if I don't do this (`val` itself is null, then you cannot evaluate `val.loading`):

> TypeError: val is null
>     apiValue ApiWrapper.vue:114

Reply:
You are absolutely correct. Thanks for that!